packages/flutter_driver/lib/src/driver/web_driver.dart

```diff
@@ -177,12 +193,14 @@ class FlutterWebConnection {
       String url,
       Map<String, dynamic> settings,
       {Duration timeout}) async {
-    // Use sync WebDriver because async version will create a 15 seconds
```

Comment:
Question: Looks like we are also switching the way we are accessing the driver object (fromExistingSession vs constructor)? Is there a specific reason for it (anything failing with a11y or screendriver)?

Reply:
The async version of fromExistingSession returns a WebDriver instance which will **NOT** have the same capabilities (in fact, it would be empty) as the original one where the capabilities plays an important role for screendriver.